### PR TITLE
Proposal for changes systemd service

### PR DESCRIPTION
**Pull request by takov751**
I have made this pull request as a proposal to restrict spotifyd in a safe environment while running it on Linux and of course on embeded systems like RPI  with running it a system service as a restricted user with only access to audio and the network. I do hope you find it useful in long term.

### CODE REVIEW

File: contrib/spotifyd.service [hunk 1]
```diff
@@ -7,7 +7,10 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-ExecStart=/usr/bin/spotifyd --no-daemon
+User=spotify
+Group=spotify
```
Comment (igorbernstein) [inline contrib/spotifyd.service:11]: These don't match the user created in README `spoty`

Comment (robinvd) [reply contrib/spotifyd.service:11]: maybe spotify or spotifyd would be a better name?

Comment (takov751) [reply contrib/spotifyd.service:11]: I have seen your comments just a moment ago . I will recreate this pull request and correct it straight away

**Comment by slondr** [comment]
Changing the systemd service file would be a breaking change for any automated installations, so this deserves careful consideration. Also, I'd like to see some copy-editing on the prose, although I'd be willing to do that myself if y'all are willing to wait until holidays are over :smile: 

I like the spirit and intention of the change, though. I think this is a good idea

File: contrib/spotifyd.service [hunk 2]
```diff
@@ -7,7 +7,10 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-ExecStart=/usr/bin/spotifyd --no-daemon
+User=spotifyd
+Group=spotifyd
+WorkingDirectory=/var/lib/spoty
```
Comment (l4l) [inline contrib/spotifyd.service:12]: forgot to fix that line as well?

```suggestion
WorkingDirectory=/var/lib/spotifyd
```

Comment (takov751) [reply contrib/spotifyd.service]: Indeed . Thank you

**Review by takov751 — COMMENTED** [review]
Corrected thank you

**Pull request closed without merging**